app/main.ts

```diff
@@ -38,6 +39,12 @@ const liveDetailsNode = document.getElementById("details");
 
 const numberPerPage: number = 7;
 
+//Add constant to limit navigation extent
```

Comment:
Instead of hardcoding the extent - since users can provide any map/data  - maybe get the map's full extent and use that? Then it'll be up to the users to ensure the web map extent is set to something appropriate for their data.

Reply:
Changed to use Map's native extent in commit cc10ed7